### DISCUSSION TITLE
read ECONNRESET error

### DISCUSSION THREAD

**Discussion by zhqpower** [Refloow/Steam-Card-Bot-PRO#41] [General]
An error occurred while getting bot Gems inventory: Error: read ECONNRESET

**Comment by Refloow** [comment]
> An error occurred while getting bot Gems inventory: Error: read ECONNRESET

Did the error crash the application or it continued working fine?
I would like to see console logs.

"ECONNRESET" means the other side of the TCP conversation abruptly closed its end of the connection. This is most probably due to one or more steam protocol errors. But there is nothing i can do about it, it happens sometimes when there is poor connection.
